contents/blog/open-source-eating-saas.md

```diff
@@ -0,0 +1,74 @@
+
+Following the open source route means you are building in a place like GitHub, which is *designed* for people to raise issues and submit feature requests. The vast majority of open source users already have a GitHub account and can just raise an issue. SaaS product feedback in the early days usually involves emails/phone calls/Slack groups. Open source has all of this available *and* their community of users are already used to leaving feedback in the repo.
+
+It's tough to measure the advantages of this stuff, but taking YCombinator launches on HackerNews, we think this is why our own launch was the most upvoted from the [W20 batch](https://bestofshowhn.com/yc-w20).. and that's with a product at the time which was only 4 weeks old. Unsurprisingly, there are now a bunch of awesome OS projects in S20 with the same or more upvotes than us :)
+
+## Your work is public 
+
+Goodbye hardcoding stuff.
+
+The complete visibility of your entire codebase is scary. But you get over it. Then you realize it *forces* you to do a much better job. It now becomes a strategic advantage to have nice code - not only can your own company get new engineers up and running fast, but you're more likely to build a healthy community if they can contribute easily.
+
+Added retention bonus: the best engineers generally don't like working on horrible code.
+
+Early on, this can make shipping a little slower, but we've found this is outweighed as a team working on your codebase can make sense of it much faster.
+
+As dumb as it sounds, it's also just more fun working transparently with users. It's a lot easier to be passionate about open source than SaaS.
+
+## More people want to work with you
+
+Life is fluid but open source gives a true sense of building something durable. It adds extra purpose to everyone's work automatically.
+
+People just want to work on something that will be [placed in a vault](https://archiveprogram.github.com/) in the Arctic, to last the next 1,000 years.
+
+This means a bigger and better hiring pipeline.
+
+It's nifty being able to say anyone in the world is welcome to use or modify what you're working on, even if they have no money to spend. It's gratifying to be offering something for free, *forever*, that lots of people can find useful.
+
+## Big companies can use it
+
+Within weeks of launching the project, we had some of the world's biggest companies using (or trying to use, and giving feedback on what went wrong!) our code.
```

Comment:
The content inside the parentheses is so long that I lost track of the word that came before them.

Maybe: "using our code (or trying to use it, and giving feedback on what went wrong!)".